code/modules/mining/minebot.dm

```diff
@@ -17,38 +17,42 @@
 /mob/living/silicon/robot/drone/mining/Initialize()
 	. = ..()
 
-	verbs += /mob/living/proc/hide
-	remove_language("Robot Talk")
-	add_language("Drone Talk", 1)
+	verbs |= /mob/living/proc/hide
+	remove_language(LANGUAGE_ROBOT)
+	remove_language(LANGUAGE_DRONE)
+	add_language(LANGUAGE_TCB, TRUE)
```

Comment:
My primary concern with that (and EAL) is that this can be used by mining drones to inform crew of a malfs directives.
Therefore I am opposed to that change, unless the Malf AI is changed to "fix" the Minig borg laws aswell.